pkg/events/events.go

```diff
@@ -110,6 +110,7 @@ func New(object interface{}, eventType config.EventType, resource, clusterName s
 
 	if objectTypeMeta.Kind == "Event" {
 		var eventObj coreV1.Event
+		var eventSeriesObj coreV1.EventSeries
```

Comment:
Where are we setting this?

Reply:
@krunalhinguu where we are setting the value of `eventSeriesObj`?
on line no 129 `event.TimeStamp = eventSeriesObj.LastObservedTime.Time` we are trying to access it. But where will `eventSeriesObj` get assigned the value?

Reply:
i thought eventSeries is providing us that value?? or are we manually do that?
@PrasadG193